File: Richest-Customer-Wealth.cpp
class Solution 
{
public:
    int maximumWealth(vector<vector<int>>& a) 
    {
        int m = 0;
        for (int i = 0; i < a.size(); i++) 
        {
            int s = 0;
            for (int j = 0; j < a[i].size(); j++) 
            {
                s += a[i][j];
            }
            if (s > m) 
            {
                m = s;
            }
        }
        return m;
    }
};

File: Richest-Customer-Wealth.py
class Solution(object):
    def maximumWealth(self, a):
        m = 0
        for i in range(len(a)):
            s = 0
            for j in range(len(a[i])):
                s += a[i][j]
            if s > m:
                m = s
        return m
